cmd/podman/secrets/create.go

```diff
@@ -0,0 +1,76 @@
+package secrets
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:   "create SECRET",
+		Short: "Create a new secret",
+		Long:  "Create a secret. Default driver is file (unencrypted).",
+		RunE:  create,
+		Args:  cobra.ExactArgs(2),
```

Comment:
You have to add a `ValidArgsFunction` for all commands. This one is used to provide suggestions on the cli if you type tab. You have to create a new one for the secrets. Look at https://github.com/containers/podman/blob/master/cmd/podman/common/completion.go